blog/_posts/2018-09-17-team-popcorn-over-and-out.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Team Popcorn says Tschüss
+layout: post
+created_at: Mon Sept 17 2018
+permalink: blog/2018-07-23-team-popcorn-over-and-out
```

Comment:
Hi @sleepypioneer ! Thanks for the PR. Loved the blog post. :+1: 

I just have minor two comments:

1. Permalink and filename should be the same. So fix the date there.

2. If you wish to send a change for Gemfile, you'll need to send a separate PR. Let's keep this PR for blog post only.

Reply:
Hi @nerdyvaishali missed this in my list of github notifications will make those changed asap!

Reply:
Hi @nerdyvaishali this was trickier for me than I thought so apologies it took a little longer. I ended up making a new PR for this => #556 I hope that is ok for you and see you Saturday! 🎉

Reply:
Hi @sleepypioneer ! Sorry for the late reply. It was nice meeting you.

Actually you can change the same PR by adding revised commits. But anyway the new PR you have sent looks good to me. So, I'm closing this one. Thanks.